man/systemd-nspawn.xml

```diff
@@ -1017,6 +1017,26 @@
         capability names and exit.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><option>--ambient-capability=</option></term>
+
+        <listitem><para>Specify one or more additional capabilities to
+        pass in the inheritable and ambient set to the program started
+        within the container. The value <literal>all</literal> is not
+        supported for this setting.</para>
+
+        <para>All capabilities specified here must be in the set
+        allowed with the --capability and --drop-capability
```

Comment:
enclose the two switches in `<option>` please. Also, suffix them with "=", as we usually do for options which accept parameters. i.e. `<option>--capability=</option>`.